src/chrome/content/rules/ilcdn.fi.xml

```diff
@@ -0,0 +1,12 @@
+<!--
+	Active mixed content:
+		- static.ilcdn.fi
```

Comment:
Since only `http://static\.ilcdn\.fi/$` seems to be affected an exclusion would be the better choice here.